### PR TITLE
Issue 426 windows version 10.0 not supported

### PR DESCRIPTION
**Pull request by SolverDude**
This is to resolve https://github.com/chef/knife-windows/issues/426

### CODE REVIEW

**Comment by irvingpop** [comment]
hey @SolverDude please try deleting the `rack` line from here: https://github.com/chef/knife-windows/blob/master/ci.gemfile#L16 and trying again.  thanks!

**Comment by SolverDude** [comment]
Alice enters gem rabbit-hole with trepidation.....

File: lib/chef/knife/bootstrap/windows-chef-client-msi.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @rem
 @rem Author:: Seth Chisamore (<schisamo@chef.io>)
-@rem Copyright:: Copyright (c) 2011-2016 Chef Software, Inc.
+@rem Copyright:: Copyright (c) 2011-2017 Chef Software, Inc.
```
Comment (spuder) [inline lib/chef/knife/bootstrap/windows-chef-client-msi.erb]: Should this be 2018 now?

**Comment by tas50** [comment]
@SolverDude This should pass if you rebase on master now

**Comment by tas50** [comment]
I'm going to close this out at this point since we have the same windows 10 support in https://github.com/chef/knife-windows/blob/master/lib/chef/knife/bootstrap/windows-chef-client-msi.erb#L90

**Pull request closed without merging**